tests/core/string/test_translation_server.h

```diff
@@ -220,4 +220,14 @@ TEST_CASE("[TranslationServer] Comparing locales") {
 
 	CHECK(res == 10);
 }
+
+TEST_CASE("[TranslationServer] Language, country and locale names") {
```

Comment:
```suggestion
TEST_CASE("[TranslationServer] Language, country, and locale names") {
```
Please use [Oxford comma](https://contributing.godotengine.org/en/latest/documentation/guidelines/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).